abci/strategies/currencypair/types.go

```diff
@@ -16,6 +16,8 @@ type OracleKeeper interface {
 	GetCurrencyPairFromID(ctx sdk.Context, id uint64) (cp slinkytypes.CurrencyPair, found bool)
 	GetIDForCurrencyPair(ctx sdk.Context, cp slinkytypes.CurrencyPair) (uint64, bool)
 	GetPriceForCurrencyPair(ctx sdk.Context, cp slinkytypes.CurrencyPair) (oracletypes.QuotePrice, error)
+	GetRemovedCPCounter(ctx sdk.Context) (uint64, error)
+	GetCPCounter(ctx sdk.Context) (uint64, error)
```

Comment:
What do we intend `CPCounter` to be used for?

Reply:
The max Amt of CP we should expect in VEs is `CPCounter`.

The max Amt of errors we should expect in VEs is `RemovedCPCounter`